src/ray/core_worker/actor_handle.h

```diff
@@ -100,6 +100,10 @@ class ActorHandle {
 
   bool ExecuteOutOfOrder() const { return inner_.execute_out_of_order(); }
 
+  bool IsActorStateSubscribed() const { return is_actor_state_subscribed_; }
```

Comment:
Sounds wired to me to add these to ActorHandle. How about adding a field `std::unordered_set<ActorID> subscribed_actors` on ActorManager?

Reply:
> 

Actually we need to store both `Subscribing` and  `Subscribed` actors.